### PR TITLE
Reviewers ranking

### PR DESCRIPTION
**Pull request by Towerthousand**

### CODE REVIEW

File: register/views.py [hunk 1]
```diff
@@ -24,6 +25,14 @@ def add_vote(application, user, tech_rat, pers_rat):
     v.save()
     return v
 
+class RankingView(LoginRequiredMixin, TemplateView):
+    template_name = 'ranking.html'
+
+    def get_context_data(self, **kwargs):
+        context = super(RankingView, self).get_context_data(**kwargs)
+        context['ranking'] = list(User.objects.annotate(vote_count=Count('vote__calculated_vote')).order_by('-vote_count')[:10].values('vote_count', 'username'))
```
Comment (casassg) [inline register/views.py]: Why only 10?

File: register/templates/ranking.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{% extends "base.html" %}
+{% load humanize %}
+{% load static %}
+{% block title %}Reviewers Ranking - {% endblock %}
+{% block head %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="{% static 'lib/bars-square.css' %}">
```
Comment (casassg) [inline register/templates/ranking.html]: you are not using (either humanize library)

File: register/views.py (hunk 1 above)
Comment (casassg) [reply register/views.py]: also why list this? values is already a list

File: register/urls.py [hunk 3]
```diff
@@ -5,5 +5,6 @@
     url(r'applications/(?P<token>\w+)/confirm$', views.ConfirmApplication.as_view(), name='confirm_app'),
     url(r'applications/(?P<token>\w+)/cancel$', views.CancelApplication.as_view(), name='cancel_app'),
     url(r'vote/$', views.VoteApplicationView.as_view(), name='vote'),
+    url(r'ranking/$', views.RankingView.as_view(), name='ranking'),
```
Comment (casassg) [inline register/urls.py:8]: add link to navbar pls

File: register/templates/ranking.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                        <tr>
+                            <th>Position</th>
+                            <th>Name</th>
+                            <th>Reviews</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for user in ranking %}
+                        <tr>
+                            <td>{{forloop.counter}}</td>
```
Comment (casassg) [inline register/templates/ranking.html]: you can use this so that its ordinal: https://docs.djangoproject.com/en/1.10/ref/contrib/humanize/#ordinal

**Review by casassg — DISMISSED** [review]
I will test on my laptop too and add feedback if missing

File: register/urls.py (hunk 3 above)
Comment (Towerthousand) [reply register/urls.py:8]: done

File: register/templates/ranking.html (hunk 4 above)
Comment (Towerthousand) [reply register/templates/ranking.html]: done

File: register/templates/ranking.html (hunk 2 above)
Comment (Towerthousand) [reply register/templates/ranking.html]: done

File: register/views.py (hunk 1 above)
Comment (Towerthousand) [reply register/views.py]: how many do you want? all?
(removed list)

**Review by casassg — APPROVED** [review]
LGTM

**Pull request merged**